rules-reviewed/eap6/websphere/websphere-xml.windup.xml

```diff
@@ -6,11 +6,11 @@
             This ruleset provides analysis of WebSphere proprietary xml files and constructs that may require individual attention when migrating to JBoss EAP 6+.
         </description>
         <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-xml,2.3.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,2.7.0.Final"/>
         </dependencies>
         <sourceTechnology id="websphere"/>
-        <targetTechnology id="eap" versionRange="[6,7)"/>
-        <tag>reviewed-2015-06-17</tag>
+        <targetTechnology id="eap" versionRange="[6,7]"/>
```

Comment:
Could you clone this into a new ruleset instead of reusing the existing one? I am concerned that the rules are a little unclear for the user if they have both EAP6 and EAP7 links in them.